build.py

```diff
@@ -112,6 +112,8 @@ def _parse_args():
     parsed.artifact_path = os.path.join(
         parsed.artifact_path, f"{parsed.model}-{parsed.quantization.name}"
     )
+    parsed.pidx2pname = dict()
```

Comment:
Perhaps we can add one line of comment mentioning that the `p` refers to parameters.

Reply:
Great point. Added one line of explanation.